static/app/views/profiling/profileSummary/index.tsx

```diff
@@ -66,6 +67,14 @@ function ProfileSummaryPage(props: ProfileSummaryPageProps) {
       search.setFilterValues('transaction', [transaction]);
     }
 
+    // there are no aggregations happening on this page,
+    // so remove any aggregate filters
+    Object.keys(search.filters).forEach(field => {
+      if (isAggregateField(field)) {
+        search.removeFilter(field);
+      }
+    });
```

Comment:
assuming we purposely want to keep the qs and not actually remove it from the url? we're just removing it from our search on this page??

Reply:
so the user can type in aggregate filters manually, and to minimize them running into invalid queries, we silently ignore in these situations in much of the product